doc/mgmt/Management Framework.md

```diff
@@ -1526,11 +1934,24 @@ Example of 'deviation' :
 ###### 3.2.2.8.2.3.2.2 Platform data is available through APIs
 
 * If constraints cannot be expressed using YANG syntax or platform data is available through feature/component API (APIs exposed by a feature to query platform specific constants, resource limitation etc.), custom validation needs to be hooked up in SONiC YANG model through custom YANG extension.
```

Comment:
Can you provide some example how to use CVL for custom validations? 

Reply:
It is as simple as specifying the custom validator function in sonic yang and implementing the same function(in go) in the custom validation file.

sonic yang:
sonic-ext:custom-validation ValidatePtp;

go function:
func (t *CustomValidation) ValidatePtp(
    vc *CustValidationCtxt) CVLErrorInfo {
}